rfps/rfp-consensus-visualization-tool.md

```diff
@@ -0,0 +1,123 @@
+
+| Milestone No. | Milestone Description | Funding | Estimated Timeframe |
+| --- | --- | --- | --- |
+| 1 | Finalize scope, a list of node API endpoints to be used, UI mockups + feedback rounds with the DevGrants team, architectural design | TBD | 2-3 weeks |
+| 2 | Implementation interoperable with a Filecoin node and User tested | TBD | 4 weeks |
+| 3 | All project deliverables are completed and added to the [Filecoin Shipyard](https://github.com/filecoin-shipyard) (website, documentation, codebase) | TBD | 2 weeks |
+| 4 | Maintenance and Upgrades during Testnet and Mainnet | TBD | 2 weeks |
+
+If a milestone is not satisfactorily met, we may not continue to fund your team for this project.
+
+## Acceptance Criteria
+
+**Acceptance criteria for Finalize scope, a list of node API endpoints to be used, UI mockups/prototype + feedback rounds with the DevGrants team, architectural design:**
+- This milestone's deliverables must represent the final design of the service and its intended interoperability with a Filecoin node API and functionality.
+- The design should address the core goals proposed in the description of this brief, with a focus on educating users about how consensus is forming in the Filecoin network.
+- The design team should engage in feedback rounds with the DevGrants team on mockups and user flows.
+- If additional tools or infrastructure would be helpful for production deployment that is clearly scoped.
+
+**Acceptance criteria for Implementation interoperable with a Filecoin node and User tested:**
+- The service must enable the functionality specified in Milestone 1 in at least one mainstream language.
+- A functioning demo of the service to be presented and testable by the PL team.
+- Additional feedback rounds with the DevGrants team as implementation progresses.
+- The tool is tested with various stakeholders and potential users for comprehensibility, utility and engagement.
+
+**Acceptance criteria for All project deliverables:**
+- All of the criteria above have been met.
+- A developer on our team can use and test the functionality of this service themselves without needing to talk to anyone.
+
+**Acceptance criteria for Maintenance and Upgrades during Testnet:**
+- The application must be usable on the Filecoin Network during Testnet and Mainnet.
+
+If a milestone is not satisfactorily met, we may not continue to fund your team for this project.
+
+## Resources
+
+#### Filecoin Network Stats and Explorers
+
+lotus Devnet
+- Grafana Network Dashboard: https://lotus-metrics.kittyhawk.wtf
+
+go-filecoin User Devnet
+- NetworkStats: https://stats.kittyhawk.wtf
+- Dashboard: http://user.kittyhawk.wtf:8010/
+- Explorer: http://user.kittyhawk.wtf:8000/
+
+> NOTE: A DevGrant RFP for a comprehensive Filecoin Block Explorer with full search has been awarded to a team in wave 1. In contrast, this RFP is specifically about exploring ways to explain and visualize Expected Consensus in Filecoin.
+
+#### Staging Devnet
```

Comment:
Why is this section called Staging Devnet? Can we rename to Additional References?